x/marker/keeper/marker.go

```diff
@@ -11,6 +13,8 @@ import (
 
 // GetAllMarkerHolders returns an array of all account addresses holding the given denom (and the amount)
 func (k Keeper) GetAllMarkerHolders(ctx sdk.Context, denom string) []types.Balance {
+	defer telemetry.MeasureSince(time.Now(), types.ModuleName, types.EventTelemetryKeyKeeperMethod, "get_all_marker_holders")
```

Comment:
I am not sure that this `types.EventTelemetryKeyKeeperMethod` is very useful here.  Seemingly if we were evaluating telemetry we would want to know operations on the `marker` module... but I don't see there being so many different types of operations that we would need to have sub category grouping here.

Reply:
Ok, got it.  I will remove them.